misc/temporary-enclave-bypass/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the web server application itself.
+# Use the -alpine variant so it will run in a alpine-based container.
+FROM public.ecr.aws/docker/library/rust:1.71.0-alpine as rust-builder
+# Base image may not support C linkage.
+RUN apk add musl-dev
+
+WORKDIR /src/
+COPY Cargo.toml Cargo.lock ./
+COPY src src
+# The '--locked' argument is important for reproducibility because it ensures
+# that we use specific dependencies.
+RUN cargo build --locked --release
+
+RUN cargo install vsock-relay
+
+FROM amazonlinux:2.0.20230207.0
+
+RUN echo "timeout=60.0" >> /etc/yum.conf
+RUN amazon-linux-extras install aws-nitro-enclaves-cli nginx1 -y && \
+    yum install aws-nitro-enclaves-cli-devel wget curl strace awscli -y && \
```

Comment:
`aws-nitro-enclaves-cli` pulls in `docker` so that doesn't need a separate install.

Seems like you don't use `wget`, `curl`, or `strace` though?

Reply:
they are nice utilities to have inside the image. nitro-shim installs the same packages